docs/docs/sourcing-from-yaml.md

```diff
@@ -0,0 +1,232 @@
+---
+title: "YAML as a source for building a site"
+---
+
+# Table of Contents
+
+- [Introduction](#Introduction)
+- [Setup](#Setup)
+- [Adding Content](#Adding-some-content)
+- [Gatsby Config](#Gatsby-configuration)
+- [Gatsby Template](#Gatsby-Template)
+- [Joining the pieces](#Joining-the-pieces)
+- [Sourcing from JSON](https://www.gatsbyjs.org/docs/using-gatsby-without-graphql/)
+
+## Introduction
+
+As you're presented with Gatsby, you'll see that you have at your disposal a myriad of ways to pull data into your website, ranging from the most used ones like [Contentful](https://www.contentful.com/), or [WordPress](https://wordpress.com/), [Drupal](https://www.drupal.org/), to the most edgier ones like [ButterCMS](https://buttercms.com/) or [GraphCMS](https://graphcms.com/).
+
+Each one using the correspondent plugin.
+
+This tutorial will help you return to the origins.
+
+Instead of pulling data using any of the available plugins for each provider mentioned, you'll create a simple fully functional Gatsby website that is sourced from a YAML file.
+
+## Prerequisites
+
+Before we go through the details and code, you should be familiar with the basics of Gatsby.
+
+Check out the [tutorial](https://www.gatsbyjs.org/tutorial/) and brush up on the [documentation](https://www.gatsbyjs.org/docs/).
+In addition to this, some knowledge of [ES6 syntax](https://medium.freecodecamp.org/write-less-do-more-with-javascript-es6-5fd4a8e50ee2) will be useful.
+
+Otherwise just skip this part and move onto the next part.
+
+## Setup
+
+You'll start by creating a new Gatsby website based on the official _hello world starter_.
+
+Open up a terminal and run the following command.
+
+```bash
+gatsby new gatsby-sourcing-YAML https://github.com/gatsbyjs/gatsby-starter-hello-world
+```
+
+After the process is complete, some additional packages are needed.
+
+Change directories to the newly created Gatsby website and issue the following command:
+
+```bash
+npm install --save js-yaml uuid
+```
+
+Or if Yarn is being used:
```

Comment:
I mentioned this in an earlier file, but we can drop the Yarn instructions from everything and allow users to focus on npm instead.